Makefile

```diff
@@ -52,7 +52,8 @@ deploy: manifests
 	cd config/default && if [ ${KFSERVING_ENABLE_SELF_SIGNED_CA} != false ]; then \
 	kustomize edit remove resource certmanager/certificate.yaml; \
 	else kustomize edit add resource certmanager/certificate.yaml; fi;
-
+	kubectl get crd inferenceservices.serving.kubeflow.org || kustomize build config/crd | kubectl create --validate=false -f -
+	kustomize build config/crd | kubectl replace --validate=false -f -
```

Comment:
Could this not be added to first part above to make it a single op?

Reply:
Also, won't the apply below for config/default do an "apply" again on the CRD as its a base of that kustomize?

Reply:
Combined to the single command, it will apply the crd again but it is pretty fast as the crd created or replaced does not have the `last-applied-annotation`.